notebooks/community/model_evaluation/automl_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #64.            problem_type=prediction_type,
</pre><p>Fine to remove "prediction_type", but please keep "problem_type="classification"". I had a mistake where the classification component required "problem_type", this will be removed in GCPC 1.0.20.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>updated.</p>